### PR TITLE
[14.0] [MIG] shopinvader_invoice

### PR DESCRIPTION
**Pull request by david-popa**

### CODE REVIEW

File: shopinvader_invoice/services/invoice.py [hunk 1]
```diff
@@ -68,39 +68,39 @@ def _get_return_invoice_schema(self):
         :return: dict
         """
         invoice_schema = {
-            "invoice_id": {"type": "integer"},
-            "number": {"type": "string"},
-            "date_invoice": {"type": "string"},
+            "move_id": {"type": "integer"},
```
Comment (acsonefho) [inline shopinvader_invoice/services/invoice.py]: IMO: To keep compatibility between version and as the link between Odoo and Json values should stay abstract, I prefer to let this schema as origin and only edit the parser to match with this structure.

File: shopinvader_invoice/services/invoice.py [hunk 2]
```diff
@@ -110,8 +110,8 @@ def _to_json_invoice(self, invoice):
         values = invoice.jsonify(parser)[0]
         values.update(
             {
-                "type_label": self._get_selection_label(invoice, "type"),
-                "state_label": self._get_selection_label(invoice, "state"),
+                "type_label": self._get_selection_label(invoice, "move_type"),
```
Comment (acsonefho) [inline shopinvader_invoice/services/invoice.py:113]: So no change here too

File: shopinvader_invoice/services/invoice.py [hunk 3]
```diff
@@ -68,39 +68,39 @@ def _get_return_invoice_schema(self):
...
         """
-        Get the parser of account.invoice
+        Get the parser of account.move
         :return: list
         """
         to_parse = [
-            "id:invoice_id",
-            "number",
-            "date_invoice",
+            "id:move_id",
```
Comment (acsonefho) [inline shopinvader_invoice/services/invoice.py]: Here you can let the `invoice_id`

**Review by acsonefho — CHANGES_REQUESTED** [review]
Only remarks about compatibility between shopinvader versions.

**Review by hparfr — APPROVED** [review]

**Comment by hparfr** [comment]
@acsonefho can you finish your review ?

**Review by acsonefho — APPROVED** [review]
Code LGTM!

**Pull request merged**